### DISCUSSION TITLE
blog/what-is-new-in-istio-1-22/

### DISCUSSION THREAD

**Discussion by giscus** [rootsongjc/rootsongjc.github.io#349] [Announcements]
# blog/what-is-new-in-istio-1-22/

探索 Istio 1.22 的新特性与实用建议，包括增量 xDS、路径模板支持及 API 升级等，助力优化服务网格管理。

https://jimmysong.io/blog/what-is-new-in-istio-1-22/

**Comment by defp** [comment]
Ambient 模式的无宕机升级仍未解决？
这个有具体的 issue 吗？

**Reply by rootsongjc** [reply]
可以关注下这个 proposal：[Safe Ztunnel Upgrades](https://docs.google.com/document/d/1TGy6ZxtMxfNoD0j9eTgMTfBeqxH-XvCi9urGxijl3Eo/edit?pli=1#heading=h.choejwi6foo2)
